macx/scripts/osxdist.py

```diff
@@ -149,9 +149,10 @@ def copy_plugins(self):
 		'''
 		print ' * Copying positional audio plugins'
 		dst = os.path.join(self.bundle, 'Contents', 'Plugins')
-		if os.path.exists(dst):
-			shutil.rmtree(dst)
-		shutil.copytree(os.path.join(options.binary_dir, 'plugins'), dst, symlinks=True)
+		if not os.path.exists(dst):
+			os.makedirs(dst)
+		for plugin in glob.glob('plugins/*.dylib'):
```

Comment:
This should've been:
```py
for plugin in glob.glob(os.path.join(options.binary_dir, 'plugins/*.dylib')):
```

CI doesn't show any issues because we use the build folder as working directory.

Reply:
The script could definitely benefit from a revamp/rewrite.

@Krzmbrzl What do you think?

Reply:
Probably true. But personally I don't think I have the time nor the motivation to do it myself :shrug:

Reply:
No worries, as long as it works we can keep it like that.